.circleci/config.yml

```diff
@@ -103,11 +103,10 @@ jobs:
           docker_layer_caching: true
       - python/install-packages:
           pkg-manager: poetry
-          cache-version: v2
```

Comment:
I think we should maybe leave this in just to be safe

Reply:
Oh okay. Thinking it might have a side effect if we go down a cache-version?

Reply:
I just don't know what's going on so refreshing all the caches seems safest. Blunt tool 🤷